docker-compose.yaml

```diff
@@ -1,59 +1,64 @@
 version: '3'
-
-# Other services are specified in .\lib\killrvideo-docker-common\docker-compose.yaml
 services:
-  node:
-    build: /Users/aleksandrvolochnev/killrvideo/node
+  # NodeJS Backend
+  backend:
+    build: .
+    volumes:
+      - .:/usr/src/app
     depends_on:
       - dse
     environment:
-      KILLRVIDEO_ETCD: "etcd:2379"
-      KILLRVIDEO_DSE_USERNAME: $KILLRVIDEO_DSE_USERNAME
-      KILLRVIDEO_DSE_PASSWORD: $KILLRVIDEO_DSE_PASSWORD
-      KILLRVIDEO_CASSANDRA_REPLICATION: $KILLRVIDEO_CASSANDRA_REPLICATION
-      KILLRVIDEO_LOGGING_LEVEL: $KILLRVIDEO_LOGGING_LEVEL
+      KILLRVIDEO_LOGGING_LEVEL: debug
```

Comment:
Not sure why
      KILLRVIDEO_DSE_USERNAME: $KILLRVIDEO_DSE_USERNAME	
      KILLRVIDEO_DSE_PASSWORD: $KILLRVIDEO_DSE_PASSWORD	
      KILLRVIDEO_CASSANDRA_REPLICATION: $KILLRVIDEO_CASSANDRA_REPLICATION
were excluded and KILLRVIDEO_LOGGING_LEVEL: debug hardcoded to debug.
All of these are dynamic especially when dealing with external clusters and our cloud deployment.

Reply:
All of the above mentioned environment variables can be set without the .env file and we do so in our cloud deployment.  Generally I don't use debug in production.  I would suggest putting those references back.  They can all be set with regular environment variables without the need for .env and this will ensure we are not missing support that the other backend's versions currently have along with the generator and webUI.